src/utils/time-utils.ts

```diff
@@ -1,8 +1,31 @@
+import { DateTime } from "luxon";
+
 /**
  * Converts inputted minutes into hours and minutes
  * @param minutes value in minutes
  * @returns inputted minute value in X h Y min format as string
  */
-export const getHoursAndMinutes = (minutes : number) : string => {
-    return `${Math.trunc(minutes / 60)} h ${(minutes % 60) * (String(minutes).startsWith("-")?-1 :1)} min`
-}
\ No newline at end of file
+export const getHoursAndMinutes = (minutes: number): string => {
+  return `${Math.trunc(minutes / 60)} h ${
+    (minutes % 60) * (String(minutes).startsWith("-") ? -1 : 1)
+  } min`;
+};
+
+/**
+ * Get time difference in days
+ *
+ * @param startDate start date
+ * @param endDate end date
+ * @returns days
+ */
+export const getTimeDifferenceInDays = (
```

Comment:
As mentioned in a previous comment it seems that the return value of this function needs to be modified according to the return value... This seems a little unclear and perhaps this further modification of the return value can actually be done in this function before it is returned.

Reply:
Moved the time difference adjustments here